packages/components/src/components/composite-checkbox/composite-checkbox.jsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Checkbox from 'Components/checkbox/checkbox.jsx';
+
+const CompositeCheckbox = ({ name, value, onChange, className, label, id, description }) => {
+    const onClickContainer = e => {
+        e.stopPropagation();
+        e.preventDefault();
+        onChange({
+            target: {
+                value: !value,
+            },
+        });
+    };
+
+    return (
+        <div
+            className={classNames('composite-checkbox', className, {
+                'composite-checkbox--active': !!value,
+            })}
+            onClick={onClickContainer}
```

Comment:
Do we want keyboard events too?

Reply:
What you have in mind?

Reply:
@reza-fs Focus handling for tabbing between the checkboxes and perhaps checking/unchecking with space